src/result.pyx

```diff
@@ -173,6 +173,13 @@ cdef class Result:
 
         return result
 
+    def write_pdb(self, filename):
+        cdef FILE *f = NULL
```

Comment:
I think we should add a few assertions with appropriate error messages. One should check that `self._c_root_node` is not null (this would happen if you use `calcCoord()`), the second should check if this comes from `calcBioPDB()`, a simple hack would be to look at the freesasa_node_name of the result node and see if it's a PDB file name (I think it will be null in the BioPDB case). A more robust, but slightly more complicated way would be to traverse down to the first atom node and check that `freesasa_node_atom_pdb_line()` doesn't return null.

Reply:
So checking the `freesasa_node_name()` for the two Structure sources (`Bio.PDB` and `freesasa.Structure`) gives the same result: `Structure`. 

I am going to see if there is a different output for `freesasa_node_atom_pdb_line()`.

Reply:
So when I try to access `freesasa_node_atom_pdb_line(atom_node)` and run the assertion `assert(atom_pdb_line is not NULL, "Atom PDB Line returned a Null Pointer")`, I get segmentation faults. 

What should I do?

Reply:
That is strange, it defaults to NULL in the C code. Can you commit what you have, so I can have a look?

Reply:
If this gets complicated we can add a doc-string to the function with a caveat for the beta and fix it in the actual release later.

Reply:
Sure here is the python code I am running. To test the different structures, I go back and forth between commenting out lines 20-23 and 27-31. 

<img width="596" alt="image" src="https://user-images.githubusercontent.com/31376682/112346604-1d539f00-8c94-11eb-8810-92f0ae4f7a14.png">

Ill push the bindings here in a minute.

Reply:
If I replace the `assert` with 
```py
if (atom_pdb_line is NULL):
    raise AssertionError("Atom PDB Line returned a Null Pointer")
```
I get the expected behavior. I have no idea why 😅 . There must be something I don't understand about Python assertions.

Reply:
you and me both! That's why they pay you the big bucks! 

ill push the changes here in a few minutes along with the updated docs.